Automattic/wp-calypso
# Site Settings: Add more settings Composing card for Jetpack sites

This PR implements After the Deadline settings for Jetpack sites in the Writing > Composing card. It also moves the current Markdown and Default Post Format settings in separate components in order to keep the main Composing card component clean. It also introduces a `setFieldValue` method to `wrapSettingsForm` that allows manually setting a specific field's value. The PR is part of #9171.

Preview (with collapsed Advanced settings):
![](https://cldup.com/D29VMOgkrO.png)

Preview (with expanded Advanced settings):
![](https://cldup.com/rszM-IjEbJ.png)

To test:

* Checkout this branch
* Go to `/settings/writing/$site` where `$site` is one of your Jetpack sites.
* Verify the default post format field works properly like before.
* Verify the Markdown field is not displayed.
* Play with the new After the Deadline settings (main toggle and sub settings) and verify they are stored and retrieved properly.
* Go to `/settings/writing/$site` where `$site` is one of your WordPress.com sites.
* Verify the default post format field works properly like before.
* Verify the Markdown field works properly like before.
* Verify the After the Deadline settings are not displayed for this site.

Styles here could benefit from #11240 - I can update the PR once it's merged.
